Realm/Realm/DatabaseTypes/RealmCollectionBase.cs

```diff
@@ -203,7 +203,7 @@ protected void AddToRealmIfNecessary(in RealmValue value)
         protected static EmbeddedObject EnsureUnmanagedEmbedded(in RealmValue value)
         {
             var result = value.AsRealmObject<EmbeddedObject>();
-            if (result.IsManaged)
+            if (result?.IsManaged == true)
```

Comment:
` == true` should not be necessary, why did you need it?

Reply:
That's because `result?.IsManaged` is a nullable bool.

Reply:
I educated myself. Makes sense.